scipy/sparse/_matrix_io.py

```diff
@@ -67,14 +68,16 @@ def save_npz(file, matrix, compressed=True):
         shape=matrix.shape,
         data=matrix.data
     )
+    if isinstance(matrix, sp.sparse.sparray):
+        arrays_dict.update(__is_array="array")
```

Comment:
It doesn't really matter, but does `__is_array=True` not work?

Reply:
I changed the key to `_is_array` and the value to `True`.